MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -1725,6 +1723,13 @@ public boolean recruitPerson(Person p, PrisonerStatus prisonerStatus, boolean gm
             }
         }
 
+        if(log){
+            formerSurname = p.getSurname().equals(formerSurname) ? "" : String.format(" (Formerly %s) ", formerSurname);
```

Comment:
Can we change this single letter variable from `p` to `person`. We're trying to move away from abbreviated or single letter variable names, so we want to clear them up whenever they're found.

Reply:
Yep, done.